code/modules/reagents/Chemistry-Reagents/Other-Reagents.dm

```diff
@@ -1008,6 +1014,13 @@
 	reagent_state = LIQUID
 	color = "#60A584" // rgb: 96, 165, 132
 
+/datum/reagent/lye
+	name = "Lye"
+	id = "lye"
+	description = "Also known as sodium hydroxide."
```

Comment:
Did we seriously not have Sodium Hydroxide in here already? Wow.

On a side note, this should really be named Sodium Hydroxide.